core/README.md

```diff
@@ -17,6 +17,14 @@ the package in your environment. The `wandb` library will automatically detect a
 pip install -U wandb wandb-core
 ```
 
+in the top of your script add the following:
+
+```python
+import wandb
+
+wandb.require("core")
```

Comment:
need to update the wording above and below re "simply" installing and uninstalling wandb-core. also, we should then just add it to the dependencies of wandb, right?

Reply:
yeah, you are right, this is what @timoffex is working on. basically create a wandb-core package that we can always include. and we are going to add it as a required dependency, this the main reason we need to have this flag back